integrated_channels/blackboard/client.py

```diff
@@ -387,6 +481,25 @@ def _delete(self, url):
             raise ClientError(response.text, response.status_code)
         return response
 
+    def _handle_course_customization_error(self, bb_course_id, bb_content_id=None):
```

Comment:
I think this method isn't necessarily (on its own) an error handling method. I think a better name for it might something like delete_course_content_and_course (or sth like it)? The context in which it is used will I guess define (e.g. during error handling) but I can see it being used outside of that context too.. just a thought

Reply:
Fair enough! I agree the method name should reflect the operations handled and not the context that the method was called within. I'm down to rename.